### PR TITLE
Emelie L and Bruna's music release project

### PR DESCRIPTION
**Pull request by waliem**
Link to my deployed project:

My reflections on how this week's project turned out:

Things I'd like to have clarified or explained in more detail:

### CODE REVIEW

File: code/src/index.css [hunk 1]
```diff
@@ -5,9 +5,230 @@ body {
...
+  color: #ffffff;
+  margin-bottom: 0;
+  text-decoration: none;
+}
+
+h3,
+a {
+  font-family: "Helvetica";
+  font-size: 18px;
+  color: #a0a0a0;
```
Comment (isomoth) [inline code/src/index.css:34]: Could these colors be replaced by variables? And other recurring elements such as the Helvetica font-family, too :)

File: code/src/components/Artists.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import React from "react"
+
+const Artists = ({ artistName, artistLink }) => {
+  return (
+    <div
+      className="links"
+      href={artistLink}
+      target="_blank"
```
[the remark this answers is not in the document]
Comment (waliem) [reply code/src/components/Artists.js:8]: oh we didn't realise that, it's fixed now. we just had target: _blank and rel: "noopener noreferrer" in the wrong place, now it works! thank you.

File: code/src/index.css (hunk 1 above)
Comment (waliem) [reply code/src/index.css:34]: Thank you for the suggestion! We will take this ideá with us for the next projects :)

File: code/src/components/Wrapper.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
+import React from "react"
+
+import Artists from "./Artists"
+import Albums from "./Albums"
+import Image from "./Image"
+
+const Wrapper = ({
```
[the remark this answers is not in the document]
Comment (waliem) [reply code/src/components/Wrapper.js:7]: It is possible to do it in App.js but we wanted to create an intermediary component to import the information and organise it in a way. that was easier for us to understand.

**Pull request closed without merging**